app/javascript/mastodon/components/poll.js

```diff
@@ -14,6 +14,7 @@ import Icon from 'mastodon/components/icon';
 
 const messages = defineMessages({
   closed: { id: 'poll.closed', defaultMessage: 'Closed' },
+  voted: { id: 'poll.voted', defaultMessage: 'You voted this answer', description: 'Tooltip of the "voted" checkmark in polls' },
```

Comment:
You missed the "for" again. Also, is the description displayed in Crowdin anywhere? If not, it's not necessary.

Reply:
I'm not sure it is. I guess we'll find out.